jest.config.ts

```diff
@@ -21,6 +21,7 @@ const config: Config = {
   collectCoverageFrom: ['src/**/*.{ts,tsx}', '!**/*.d.ts', '!**/*constants.ts'],
   coveragePathIgnorePatterns: ['/node_modules/', '/tests/.*\\.(ts|js)$'],
   moduleNameMapper: {
+    '^(\\.{1,2}/.*)\\.js$': '$1',
```

Comment:
For future viewers: asked David about this line and it's to handle ts-jest dynamic imports temporarily, lol